src/HotReload.Listener/Startup.fs

```diff
@@ -11,7 +11,7 @@ type Startup() =
     // This method gets called by the runtime. Use this method to add services to the container.
     // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
     member this.ConfigureServices(services: IServiceCollection) =
-        services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2) |> ignore
+        services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1) |> ignore
```

Comment:
had to use netcoreapp2.1 for some reason, which is why this flipped back

Reply:
> so the bad stuff was that I had to manually make a change to portacode to let it run under netcoreapp, so keep an eye on that.

Was this just skipping over the exception on loading `System.IO.Pipelines` or did you have to make other changes?